cybershell/fingerprint.py

```
@@ -0,0 +1,649 @@
+            futures = []
+            for endpoint in endpoints:
+                future = executor.submit(self._check_endpoint, base_url + endpoint)
+                futures.append((endpoint, future))
+            
+            for endpoint, future in futures:
+                try:
+                    result = future.result(timeout=5)
+                    if result:
+                        fingerprint.raw_signals[f'endpoint_{endpoint}'] = result
+                        self._analyze_endpoint_result(fingerprint, endpoint, result)
+                except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)